### ISSUE TITLE
Multi-output model support

### ISSUE THREAD

**Issue by gokceneraslan** [Avsecz/keras-hyperopt#4]
As far as I understand from expressions such as `np.arange(min(n_train, train[1].shape[0]))` in `test_fn`, targets must be list/tuple right now, although they can be dict in Keras to allow models with multiple outputs. So it would be nice to add dict target support, at some point. So I am adding this issue just as a TODO item.

**Comment by Avsecz** [comment]
agree. https://github.com/Avsecz/keras-hyperopt/blob/master/kopt/model_data.py#L76 needs to be generalized as for the inputs.
